lib/griddler/mandrill/adapter.rb

```diff
@@ -59,7 +59,17 @@ def full_email(contact_info)
         end
       end
 
+      def add_base64_field(event, key, default_value)
+        if event[key]
+          event[key].each_value do |value| 
+            value[:base64] = default_value if !value.has_key?(:base64)
+          end
+        end
+      end
+
       def attachment_files(event)
+        add_base64_field(event, :images, true)
```

Comment:
I disagree. You can still keep the logic in a separate method but the inclusion/exclusion of the base64 attribute is an actual part of processing each file. We don't need to double the time complexity here. The logic around processing files may need to be refactored to maintain a single responsibility.